README.md

```diff
@@ -1,8 +1,10 @@
 # kc15
 Public release of the ancient (but possibly revivable) KC1 project, regeneration no. 5
+or "KC15", for short.
 
 TODO
-1. Verify files can actually be built on a modern system 
+
```

Comment:
A trailing space was needed to make the list into a list with automatic linebreaks.

I feel poorer in time for having occupied some of my head with that data.